gamefixes-steam/950670.py

```diff
@@ -0,0 +1,53 @@
+"""Game fix for Gothic Playable Teaser"""
+
+from pathlib import Path
+from protonfixes import util
+from protonfixes.logger import log
+
+
+def main() -> None:
+    """Performance fixes and cleanup of excessive logs"""
+    modify_settings()
+    clear_logs()
```

Comment:
Hard to say if we should do this. Clearing the logs would be a plus, but wouldn't make the game more playable compared to the `modify_settings`.

Reply:
Personally, though, I would want the logs removed to save some space. 1GB+ of generated logs would be a problem for me and I'd hate it.

Reply:
My thinking was to "fix" what the game was doing wrong. So it could be seen as a kind of "game fix." 😊 

It's pretty unlikely to cause any harm, but it might also be seen as outside the scope of what we're aiming for.